### ISSUE TITLE
Request delist Hitbtc exchange

### ISSUE THREAD

**Issue by Curro666** [ccxt/ccxt#14726]
Hello,
I have been waiting for more than a year to be able to get several coins from the Hitbtc exchange without success. This is why I request that you be delisted from ccxt to protect its users.
Among the currencies that cannot be withdrawn are steem and dogecoin. Dogecoin was at very high points last year and they did not allow withdrawal, yes, they perfectly allow income in dogecoin so the wallet is perfectly synchronized and there are plenty of excuses.
The reason why I have opened this thread is because yesterday I saw an old post that cointiguer was delisted for similar problems.
I hope the ccxt team will contact them and verify the information I just provided. If you need a screenshot or more information I can provide it without problem.
BR

**Comment by Dan-krm** [comment]
Thanks for your request, difficulty withdrawing funds from Hitbtc is a known issue and can generally be resolved by converting to a different currency before withdrawing or adding more funds to your account to reach the minimum amount required to withdraw. I don't think this necessarily means that Hitbtc should be delisted but ccxt considers all feedback!

**Comment by Curro666** [comment]
Yes, when the price is normal nots a problem but if you see the price of doge for example in hitbtc DOGE/BTC are 0.00000080 and the real price is 0.00000332.
I personally believe that Hitbtc is a scam exchange. Very much users have the same problem in other coins. 
In the case of doge it is clearly a scam because they perfectly allow you to make deposits in the wallet and it is well reflected in the balance but they do not let you withdraw them. I've been waiting for more than a year to get my coins out of there.
The support service never responds to anything new, they only limit themselves to giving you the url where you can check the status and that in the cases in which they respond, many times not even that.
The price in usdt touched 0.70 we are talking about a total of about $16,000 and as you will understand I am not willing to sell them for $437 which is the total at the price of 0.01 that is on hitbtc

**Comment by pier0074** [comment]
Hitbtc has a price problem with DOGE since a long time, reason why they have deactivated withdrawal from this currency.
You can check the status of everything from [this page](https://hitbtc.com/system-monitor). As far as I know DOGE withdrawal has been deactivated for more than a year.

**Comment by Curro666** [comment]
> Hitbtc has a price problem with DOGE since a long time, reason why they have deactivated withdrawal from this currency. You can check the status of everything from [this page](https://hitbtc.com/system-monitor). As far as I know DOGE withdrawal has been deactivated for more than a year.

Yes, everyone sends to that address to check (https://hitbtc.com/system-monitor) , to waste time. As you can see in that address, if you go to DOGE you will see how withdrawals have been since May 28, 2021 without allowing withdrawals, yes, curiously, they continue to accept deposits. The last deposit was today. As you will see, the wallet is still active and perfectly synchronized, so it is clearly seen that it is not a maintenance problem or the like. We have been waiting for more than a year to get our coins back and they certainly have no intention of doing so.
The same goes for ETC.

**Comment by ttodua** [comment]
Exchange not allowing to withdraw DOGE for so long period, is very shady misbehavior. I can't say that it is reason for exchange to get delisted, but neither small misdeed. 
We should keep an eye for other reports too.

**Comment by carlotestor** [comment]
Sorry you're stuck in this situation — but this is outside what CCXT can address. CCXT is an open-source library that provides a unified programming interface to exchange APIs; it isn't a listing service, custodian, or intermediary, has no commercial relationship with HitBTC, and no ability to influence or arbitrate withdrawals. Removing the HitBTC integration wouldn't return your funds and would only break tooling for users who still transact there.

Where to actually escalate:
- Formal complaint to the exchange's registered entity and, if it exists, its regulator in that jurisdiction.
- Chargeback/legal routes if fiat was involved.
- Public reporting sites and consumer-protection bodies, which do influence exchange behaviour.

For what it's worth, `hitbtc` remains in CCXT, so the practical thing you *can* do programmatically is check status before committing funds anywhere:

```python
exchange = ccxt.hitbtc()
currencies = exchange.fetch_currencies()
print(currencies['DOGE']['active'], currencies['DOGE']['withdraw'], currencies['DOGE']['deposit'])
```
`withdraw: False` on a currency is the signal to avoid depositing it — worth wiring into any bot as a pre-flight check across exchanges.

Closing thought: CCXT does track reports like yours informally, as @ttodua noted, but delisting decisions are driven by API viability rather than fund disputes.
